projects/Amlogic/kodi/advancedsettings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<advancedsettings>
+  <cputempcommand>cputemp</cputempcommand>
+  <gputempcommand>cputemp</gputempcommand>
```

Comment:
I think it is better to symlink `gputemp`/`cputemp` then this file can be removed.